rb/lib/selenium/webdriver/common/driver_extensions/takes_screenshot.rb

```diff
@@ -32,6 +32,11 @@ module TakesScreenshot
         #
 
         def save_screenshot(png_path)
+          extension = File.extname(png_path).downcase
+          if extension != 'png'
```

Comment:
This should be `'.png'` as `File.extname` also gives the period. Can you add a spec for the complement testing that it doesn't warn for a file name with `.png`?